docs/providers/documentation/parseable-provider.mdx

```diff
@@ -0,0 +1,46 @@
+---
+title: "Bash"
```

Comment:
Looks like a mistake, it's parseable-provider.mdx 0_o

Reply:
Shit, my mistake

Reply:
Lol, no worries!